### PR TITLE
assignment 1

### PR DESCRIPTION
**Pull request by andu470**
Final assignment1 - a little problem with parsing loggedIn id

### CODE REVIEW

**Comment by andu470** [comment]
Desigur, nu final project ar fi trebui sa se numeasca pull request-ul, ci final assignment1

File: Assignment 1/Rotaru Andrei/src/main/java/controller/AdminController.java [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+import java.util.List;
+
+public class AdminController {
+    private final AdminView adminView;
+    private final LoginView loginView;
+
+    private final UserService userService;
+    private final ActivityService activityService;
+
+    private final TableProcessing tableProcessing;
```
Comment (Alex-Cosma) [inline Assignment 1/Rotaru Andrei/src/main/java/controller/AdminController.java:26]: Could just be a service
Definitely wouldn't put it in the `controller` package

File: Assignment 1/Rotaru Andrei/src/main/java/controller/SessionController.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
+package controller;
+
+public abstract class SessionController {
```
Comment (Alex-Cosma) [inline Assignment 1/Rotaru Andrei/src/main/java/controller/SessionController.java:3]: Very interesting. I like this

**Review by Alex-Cosma — COMMENTED** [review]
But overall good work

**Pull request merged**